js/platform-window.js

```diff
@@ -1,85 +1,125 @@
 import { html, render } from 'https://unpkg.com/lit-html@1.0.0/lit-html.js';
+import { getTemplates, storeTemplate, getTemplateByName, onStoreUpdate } from './template-store.js';
 
 window.addEventListener('DOMContentLoaded', () => {
-    const containerId = 'layout-container';
 
     fin.me.on('layout-ready', async () => {
         // Whenever a new layout is ready on this window (on init, replace, or applyPreset)
         const { settings } = await fin.Platform.Layout.getCurrentSync().getConfig();
         // determine whether it is locked and update the icon
         if(settings.hasHeaders && settings.reorderEnabled) {
-            document.getElementById('lock-button').firstChild.src = unlockedSvg;
+            document.getElementById('lock-button').firstChild.src = UNLOCKED_SVG;
         } else {
-            document.getElementById('lock-button').firstChild.src = lockedSvg;
+            document.getElementById('lock-button').firstChild.src = LOCKED_SVG;
         }
     });
     // Before .50 AI version this may throw...
-    fin.Platform.Layout.init({containerId});
+    fin.Platform.Layout.init({CONTAINER_ID});
```

Comment:
Does the api expect this casing?

Reply:
Guess it only worked because I did not remove the id from the element.